### ISSUE TITLE
Show road surface information on long tap

### ISSUE THREAD

**Issue by DzmitryiK** [organicmaps/organicmaps#3006]
Now, on long press on road, there is only information about road name (if available), and road number (also, if available). 
I propose to add information about surface (tag `surface=*`), which can be useful while planning routes. 

For example, I could drive on a gravel road in a car, but I would not like to drive on bare ground, especially in spring/summer. Another example - I could ride a bike on bare ground, but on gravel it would be hard and I want to avoid it. These are the cases where the Avoid Unpaved Roads option is not enough.

The implementation could be similar to showing cafe cuisines:
![image](https://user-images.githubusercontent.com/63190498/180444947-4e0e365f-6fab-428f-b4e3-56807f0e356b.png)
![image](https://user-images.githubusercontent.com/63190498/180445289-59ba712d-1787-435f-aa94-dd069f9930c3.png)
![image](https://user-images.githubusercontent.com/63190498/180445676-0d000c88-d54e-45fe-83a1-f554d0a48024.png)
![image](https://user-images.githubusercontent.com/63190498/180446814-32d6975c-8edc-4070-80ee-223eab25d48a.png)

**Comment by biodranik** [comment]
This information is more useful when building a route )

**Comment by DzmitryiK** [comment]
> This information is more useful when building a route )

You are right, but as I see, in the near future this won't be available (showing road surface in route building). 
It seems to me that this is easier and faster to make, than to add full support of road surface into the route building (which also will be great and I am waiting for that). This can also help in further implementation of route construction.

**Comment by biodranik** [comment]
Road surface is probably not saved atm, right @vng ?

**Comment by vng** [comment]
It transformed into 4 options:
paved_good
paved_bad
unpaved_good
unpaved_bad

**Comment by vng** [comment]
https://github.com/organicmaps/organicmaps/issues/2704

**Comment by DzmitryiK** [comment]
Maybe it is possible to add surface information in the process of transformation to some kind of new field? Or it will affect map size?

**Comment by biodranik** [comment]
Map size and processing speed will be affected with a "straightforward" solution. A careful design and implementation are needed. The question is "does it worth it for this feature"?

It probably can be implemented similarly to OSMand, but then the question is "why is it more important than other missing features in OM"?

**Comment by DzmitryiK** [comment]
It does not worth it JUST for this feature) But if you look at this as a part of new route construction algorithm, that will need coverage information, or even fixing #2704 mentioned above - it's becoming not so useless anymore.

OSMand does not have information about surface on long tap - only a graph in route construction and a layer. Both are not very useful, because a graph is shown only when route is already built and a layer makes map quite messy and not very clear without a legend.

About importance - yes, it may be not so important, as for example track recording. But as I see improving route construction is also one of your priorities. If it will require information about road surface - consider making it visible - maybe the way I propose, maybe another.
